src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/documents/DocumentApiClient.kt

```
@@ -65,18 +63,27 @@ class DocumentApiClient(@param:Qualifier("oauthDocumentApiClient") val documentA
     documentApiClient.delete()
       .uri("/documents/$documentUuid")
       .header("Service-Name", "create-and-vary-a-licence-api")
-      .accept(MediaType.APPLICATION_PDF)
+      .accept(MediaType.APPLICATION_JSON)
       .retrieve()
       .onStatus(HttpStatusCode::isError) { response ->
-        response.bodyToMono<String>().map { body ->
-          error(
-            "Error deleting document (UUID=$documentUuid, StatusCode=${
-              response.statusCode().value()
-            }, Response=$body)",
-          )
```

This would fail silently if there was no response payload